src/goto-checker/symex_bmc_incremental_one_loop.cpp

```diff
@@ -0,0 +1,135 @@
+/*******************************************************************\
+
+Module: Incremental Bounded Model Checking for ANSI-C
+
+Author: Peter Schrammel, Daniel Kroening, kroening@kroening.com
+
+\*******************************************************************/
+
+#include <limits>
+
+#include <util/source_location.h>
+#include <util/xml.h>
+
+#include "symex_bmc_incremental_one_loop.h"
+
+symex_bmc_incremental_one_loopt::symex_bmc_incremental_one_loopt(
+  message_handlert &message_handler,
+  const symbol_tablet &outer_symbol_table,
+  symex_target_equationt &target,
+  const optionst &options,
+  path_storaget &path_storage,
+  guard_managert &guard_manager)
+  : symex_bmct(
+      message_handler,
+      outer_symbol_table,
+      target,
+      options,
+      path_storage,
+      guard_manager),
+    incr_loop_id(options.get_option("incremental-loop")),
+    incr_max_unwind(
+      options.is_set("unwind-max") ? options.get_signed_int_option("unwind-max")
+                                   : std::numeric_limits<unsigned>::max()),
+    incr_min_unwind(
+      options.is_set("unwind-min") ? options.get_signed_int_option("unwind-min")
+                                   : 0)
+{
+  ignore_assertions =
+    options.get_bool_option("ignore-properties-before-unwind-min");
```

Comment:
Perhaps a question for @peterschrammel 

> When was this option introduced and where is it documented?

https://github.com/diffblue/cbmc/pull/4361#discussion_r292647453

Reply:
It has been introduced as part of this PR. Maybe it hasn't been documented as part of thIs PR, though.

Reply:
Hmm, seems to be documented here @tautschnig : https://github.com/diffblue/cbmc/pull/5217/files#diff-e58a25459f0e8ca5a6c351b1d9d75167R225

Reply:
```suggestion
    incr_min_unwind >= 2 && options.get_bool_option("ignore-properties-before-unwind-min");
```
@peterschrammel or at least this way it was in 5.4.

Reply:
Any observable effect of adding this?

Reply:
Well since the test(s) in question do not set the `unwind-min` I assume it is 1 by default. So the ignoring would not even occur. So yes, the test passes, but rather trivially.

Reply:
On the other hand, the test is still wrong if you set the `unwind-min` (to 10 for example) even with the suggested change.